client/app/_layout.js

```diff
@@ -1,27 +1,31 @@
-import { Slot, SplashScreen } from 'expo-router';
-import { Platform } from 'react-native';
+import { Slot } from 'expo-router';
+
+import { Platform, View } from 'react-native';
+
+import Navigation from '../screens/Navigation';
+
 import { Provider } from 'react-redux';
 import { PersistGate } from 'redux-persist/integration/react';
+
 import { store, persistor } from '../store/store';
+
 import { AuthProvider } from '../context/auth';
 import { ThemeProvider } from '../context/theme';
+import FlashMessage from 'react-native-flash-message';
 import Footer from '../components/footer/Footer';
-import { NetworkStatusProvider } from '../context/NetworkStatusProvider';
-import Header from '../screens/header';
 
 export default function HomeLayout() {
   return (
     <Provider store={store}>
       <PersistGate loading={null} persistor={persistor}>
-        <AuthProvider>
-          <NetworkStatusProvider>
-            <ThemeProvider>
-              <Header />
-              <Slot />
-              {Platform.OS === 'web' ? <Footer /> : null}
-            </ThemeProvider>
-          </NetworkStatusProvider>
-        </AuthProvider>
+        {/* <AuthProvider> */}
+        <ThemeProvider>
+          <FlashMessage position="top" />
```

Comment:
Flash message in here ? 

Also so it looks like now the layout in (app) is where we will handle session access instead of in root, right ?